draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
```

Comment:
> if you set recvonly direction and call createOffer, is a=msid still present

No; creating a sendrecv transceiver and changing it to recvonly should have the same effect as just creating it as recvonly.

> If not, and you change the track with replaceTrack and call setDirection("sendrecv"), which MSID is used?

Currently, the MSID from the track, which I don't see a problem with. Basically, the first time "a=msid" is generated is when it gets locked in.